bindings/java/java/src/main/java/org/ethereum/evmc/Host.java

```diff
@@ -0,0 +1,97 @@
+package org.ethereum.evmc;
+
+import static org.ethereum.evmc.EvmcVm.getContext;
+
+import java.nio.ByteBuffer;
+
+/**
+ * The Host interface.
+ *
+ * <p>The set of all callback functions expected by VM instances.
+ */
+public final class Host {
```

Comment:
Will that work if this class is package private? It seems like we don't need to expose it to the user of this API.